gdal/build.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
+./configure --with-python --prefix=$PREFIX \
+--with-geos=$PREFIX/bin/geos-config \
+--with-static-proj4=$PREFIX \
+--with-hdf5=$PREFIX \
+--with-hdf4=$PREFIX \
+--with-netcdf=$PREFIX \
+--with-xerces=$PREFIX \
+--with-sqlite3=$PREFIX \
```

Comment:
since we are setting "curl" as a requirement, perhaps "--with-curl=$PREFIX/bin/curl-config" should be added?